plugins/README.md

```diff
@@ -36,11 +36,10 @@ http://localhost:4723/inspector
 ## Development
 
 1. `git clone` this repositiry
-2. `appium plugin install --source=local /path/to/appium-inspector/plugins`
-3. Update the plugin content with `npm run build:plugin` in `/path/to/appium-inspector`
-4. Start Appium with `appium --use-plugins=inspector --allow-cors`
-
-This plugin only needs the `appium` server as a `peerDependencies`.
+2. Run `npm install` in `/path/to/appium-inspector/plugins`
+3. `appium plugin install --source=local /path/to/appium-inspector/plugins`
+4. Update the plugin content with `npm run build:plugin` in `/path/to/appium-inspector`
+5. Start Appium with `appium --use-plugins=inspector --allow-cors`
```

Comment:
Can this be moved to #Usage section?

Reply:
Do you mean in addition to the existing Usage's 1)?